retroarch.c

```diff
@@ -90,6 +86,7 @@
 #endif
 
 #include "autosave.h"
+#include "command.h"
```

Comment:
Should this check `#ifdef HAVE_COMMAND` or is that handled elsewhere?

Reply:
I honestly have no idea other than no other file seems to be doing that so its probably not needed. Given that it was included in that file twice it was probably missed because of that conditional...